client/src/components/CardContainer/CardContainer.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {NavLink} from "react-router-dom";
+import {Swiper, SwiperSlide} from "swiper/react";
+import {Navigation} from 'swiper'
+import CardItem from '../CardItem/CardItem';
+import styles from './CardContainer.module.scss';
+
+
+import "swiper/css";
```

Comment:
please, fix this, because it is wrong import

https://namespaceit.com/blog/module-not-found-cant-resolve-swipercss